ui/pages/confirmation/components/confirmation-network-switch/index.scss

```diff
@@ -1,11 +1,37 @@
 .confirmation-network-switch {
+  --icon-size: 64px;
+
+  display: flex;
+  justify-content: center;
+  margin-top: 8px;
```

Comment:
All components from the component library allow style utility props for display, flexbox, margin and padding we should use those as much as possible because it reduces the need to write CSS and reduces the amount of CSS we have to load in the browser